src/main/java/com/google/devtools/build/lib/rules/cpp/CcCompilationHelper.java

```diff
@@ -800,6 +800,11 @@ public CompilationInfo compile() throws RuleErrorException {
     outputGroupsBuilder.putAll(
         CcCommon.createSaveFeatureStateArtifacts(
             cppConfiguration, featureConfiguration, ruleContext));
+    outputGroupsBuilder.put(
+        "index_store",
+        NestedSetBuilder.<Artifact>stableOrder()
+            .addAll(ccCompilationOutputs.getIndexStoreFiles()).build()
```

Comment:
what if there aren't any?

Reply:
Then the output group is an empty depset